debian/changelog

```diff
@@ -1,3 +1,10 @@
+matrix-synapse-py3 (1.24.0+nmu1) UNRELEASED; urgency=medium
+
+  * Update dependencies to account for the removal of the transitional
```

Comment:
... And fix the tests on 3.5/3.6?

Reply:
I guess that's not really a major user-facing change.

Reply:
Yeah. And it's really only _our_ Docker-based build environments that broke, since you have to have both an older Python *and* `LANG=C` which should be very rare.

For example, our CI uses the "official" Docker images for Python, which set `LANG=C.UTF-8`, and any developer machine is likely to have a "real" locale set.